src/throttledStore.tsx

```diff
@@ -34,7 +35,7 @@ export const createThrottledStore = (
     requestAnimationFrame: Window['requestAnimationFrame'],
     cancelAnimationFrame: Window['cancelAnimationFrame']
 ): ThrottledStore => {
-    const store = createStore(reducer)
+    const store = createStore(reducer, devToolsEnhancer({ name: 'repluggable' }))
```

Comment:
let's conditionally wrap this with an option like: `disableCheckCircularDependencies`, like: `enableReduxDevtoolsExtension` because if we dont users who have this extension will get data on the store 
the value we can send from `responsive-editor` and make true only in debug mode

Reply:
added